packages/amplify-category-auth/index.js

```diff
@@ -42,6 +42,9 @@ async function add(context) {
         service: resultMetadata.service,
         providerPlugin: resultMetadata.providerName,
       };
+      if (context.amplify.auth && context.amplify.auth.dependsOn) {
```

Comment:
How do we add context.amplify.auth? Is this the amplifyMeta?

Reply:
I can't recall why we were adding this to the context as opposed to reading the parameters file, but I think it may be because we were previously updating the parameters file differently.  At this point there is no need for it, so I refactored to read the auth params file directly.